app.rb

```diff
@@ -1,8 +1,44 @@
 require 'sinatra/base'
+require_relative './lib/player.rb'
+
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
+
   get '/test' do
```

Comment:
I feel like this could be removed for the final solution, if it was used for testing MVC

Reply:
Don't I still need sessions to get the params?